src/parser/classes/livechat/items/LiveChatPaidMessage.ts

```diff
@@ -38,13 +38,13 @@ class LiveChatPaidMessage extends YTNode {
       id: data.authorExternalChannelId,
       name: new Text(data.authorName),
       thumbnails: Thumbnail.fromResponse(data.authorPhoto),
-      badges: Parser.parseArray<MetadataBadge>(data.authorBadges, MetadataBadge),
```

Comment:
There's probably a good reason why this was typed `MetadataBadge`. Maybe opt for using `Parser.parseArray<LiveChatAuthorBadge | MetadataBadge>(data.authorBadges, [MetadataBadge, LiveChatAuthorBadge])` instead?

Reply:
I haven't run into any case where this was necessary over the past 12 hours, but having a "failsafe" is never a bad idea, you got a point.
Let me add that to the other classes as well just in case.